gpustack/server/controllers.py

```diff
@@ -167,10 +167,16 @@ async def _reconcile(self, event: Event):
                     return
 
                 if event.type == EventType.DELETED:
-                    await sync_replicas(session, model, self._config)
+                    # trigger model replica sync
+                    event_bus.publish(
+                        Model.__name__.lower(),
+                        Event(type=EventType.UPDATED, data=model),
+                    )
+                    return
```

Comment:
The function "ensure_model_instance_mcp_bridge" also performs operations when the event is "delete".